### PR TITLE
Feature/open source

### PR DESCRIPTION
**Pull request by phlndrwd**
Addition of 3-Clause BSD license file and change to source headers.

Test outputs: http://fcm1/cylc-review/taskjobs/punderwo/?suite=monio_open_source_03

Completely addresses issue #1.

### CODE REVIEW

**Comment by matthewrmshin** [comment]
I guess, after this change, we can submit a ServiceNow request to turn this repo into a public one?

**Comment by phlndrwd** [comment]
> I guess, after this change, we can submit a ServiceNow request to turn this repo into a public one?

I'm still waiting for the formal approval just now. @DavidSimonin asked me to get this ready for when that's given. That's why the PR is still current in draft.

But yes, if that's the process for achieving this once approval is given.

File: README.md
Comment (mo-joshuacolclough) [inline README.md]: Minor typo (same occurs under **Writing State Files**)
```suggestion
Writing of an LFRic-compatible, time-independent, increment file is dependent on geometry data and other metadata being available at the resolution you intend to write. These will be available if MONIO has already been used to read LFRic-compatible data at the same resolution you intend to write (see the read functions described above). If MONIO has not been used for reading, writing will first require that geometry and metadata are copied from an appropriate input file using the following call:
```

**Review by mo-joshuacolclough — COMMENTED** [review]
Nice, looks good

**Comment by phlndrwd** [comment]
> Nice, looks good

Thanks Josh!

**Review by mo-joshuacolclough — APPROVED** [review]

**Comment by phlndrwd** [comment]
Hi @yaswant. Following authorisation from Glenn, this PR has been merged now. Do you know where I place the ServiceNow request @matthewrmshin referred to above to make this repo publicly accessible?

**Comment by yaswant** [comment]
> Hi @yaswant. Following authorisation from Glenn, this PR has been merged now. Do you know where I place the ServiceNow request @matthewrmshin referred to above to make this repo publicly accessible?

I don't think there is a specific SN category for this, so you can either email SAM (Software Asset Management) or open a SN request under "Can't Find What You're Looking For" category with attn to SAM.

**Pull request merged**
